arch/x86/core/msr.c

```diff
@@ -75,3 +75,18 @@ u64_t _MsrRead(unsigned int msr)
 
 	return ret;
 }
+
+/* FIXME: find better location */
+#ifdef CONFIG_JAILHOUSE_X2APIC
+#define MSR_X2APIC_BASE 0x00000800
+
+u32_t read_x2apic(unsigned int reg)
```

Comment:
could maybe make these inline functions in kernel_arch_func.h

Reply:
okay